Rubberduck.Parsing/ComReflection/ComVariant.cs

```diff
@@ -39,7 +42,6 @@ public class ComVariant : IEquatable<ComVariant>
             {VarEnum.VT_R8, Tokens.Double}
         };
 
-
         [StructLayout(LayoutKind.Sequential)]
         private struct Variant
```

Comment:
FYI, we seem to have a similar definition for `Variant` here: 
https://github.com/rubberduck-vba/Rubberduck/blob/next/Rubberduck.VBEEditor/ComManagement/TypeLibs/TypeLibsSupport.cs#L69

Probably want to unify the COM structures.